src/sxmlc.c

```diff
@@ -1752,7 +1826,11 @@ int XMLDoc_parse_file_SAX(const SXML_CHAR* filename, const SAX_Callbacks* sax, v
 	if (sax == NULL || filename == NULL || filename[0] == NULC)
 		return FALSE;
 
+#ifdef _MSC_VER
+	fopen_s(&f, filename, fmode);

```

Comment:
In my understanding `fopen_s()` is not available everywhere, so why use it here?

Reply:
Same reason as above, it's because Microsoft replaced some standard c functions by their own implementation, and they strongly push to use them instead of standard versions, for security reasons. Especially while you try to compile in 64 bit.